lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php

```diff
@@ -28,7 +28,7 @@
  *
  * @author Daniel Leech <daniel@dantleech.com>
  */
-class QueryBuilder extends AbstractNode
+class QueryBuilder extends QBConstants
```

Comment:
Same here. Did you chose to use an alias for a reason? A renaming in a future major version maybe?